cumulus/parachains/runtimes/assets/asset-hub-westend/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -220,4 +220,14 @@ impl<T: frame_system::Config> WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(9))
 			.saturating_add(T::DbWeight::get().writes(4))
 	}
+
+	pub fn exchange_asset() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `159`
+		//  Estimated: `6196`
+		// Minimum execution time: 87_253_000 picoseconds.
+		Weight::from_parts(88_932_000, 6196)
+			.saturating_add(T::DbWeight::get().reads(9))
+			.saturating_add(T::DbWeight::get().writes(4))
+	}
```

Comment:
This should be benchmarked eventually, but it's fine like this for Westend - it will get benchmarked outside of this PR at a later point